modules/cnf-topology-aware-lifecycle-manager-backup-feature.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+// Epic CNF-3901 (CNF-2133) (4.11), Story TELCODOCS-339
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_content-type: PROCEDURE
+[id="talo-backup-start_and_update_{context}"]
+= Creating a ClusterGroupUpgrade CR with backup
+
+For {sno}, you can create a backup of a deployment before an upgrade. If the upgrade fails you can use the backup to return the system to its preupgrade state.
```

Comment:
For {sno}, you can create a backup of a deployment before an upgrade. If the upgrade fails you can use the upgrade-recovery.sh script generated by TALM to return the system to its preupgrade state.

I think it's important here to note that TALM creates a recovery script which simplifies the recovery process. The simple recovery process is an important part of the backup feature in TALM.

Reply:
ok